cmd/nightfalldlp/main.go

```diff
@@ -90,10 +90,10 @@ func CreateDiffReviewerClient() (diffreviewer.DiffReviewer, error) {
 		return github.NewAuthenticatedGithubService(githubToken), nil
 	case usingCircleCi():
 		githubToken, ok := os.LookupEnv(githubTokenEnvVar)
-		if !ok {
-			return nil, fmt.Errorf("could not find required %s environment variable", githubTokenEnvVar)
+		if !ok || githubToken == "" {
+			return circleci.NewUnauthenticatedCircleCiService(), nil
```

Comment:
It feels like we should do an info log if the github token is missing, to indicate that no findings will be directly posted to github, thoughts?

Reply:
sure, that makes sense